packages/integration/lib/src/widgets/activity/activity_widget.dart

```diff
@@ -0,0 +1,361 @@
+import 'package:flutter/material.dart';
+import 'package:likeminds_feed_flutter_core/likeminds_feed_core.dart';
+import 'package:likeminds_feed_flutter_ui/packages/expandable_text/expandable_text.dart';
+import 'package:video_player/video_player.dart';
+
+class LMFeedActivityWidget extends StatefulWidget {
+  const LMFeedActivityWidget({
+    super.key,
+    required this.uuid,
+    this.postWidgetBuilder,
+    this.commentWidgetBuilder,
+    this.appBarBuilder,
+  });
+  final String uuid;
+
+  final LMFeedPostWidgetBuilder? postWidgetBuilder;
+  final LMFeedPostCommentBuilder? commentWidgetBuilder;
+  final LMFeedPostAppBarBuilder? appBarBuilder;
+
+  @override
+  State<LMFeedActivityWidget> createState() => _LMFeedActivityWidgetState();
+}
+
+class _LMFeedActivityWidgetState extends State<LMFeedActivityWidget> {
+  late Future<GetUserActivityResponse> _activityResponse;
+
+  @override
+  void initState() {
+    loadActivity();
+    super.initState();
+  }
+
+  void loadActivity() async {
+    final activityRequest = (GetUserActivityRequestBuilder()
+          ..uuid(widget.uuid)
+          ..page(1)
+          ..pageSize(10))
+        .build();
+    _activityResponse = LMFeedCore.client.getUserActivity(activityRequest);
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    LMFeedThemeData feedTheme = LMFeedTheme.of(context);
+    return Scaffold(
```

Comment:
This widget should be outside of scaffold widget

Reply:
Done